### PR TITLE
4th post birder series do not review yet

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

**Comment by maelle** [comment]
Post preview https://deploy-preview-245--ropensci.netlify.com/blog/2018/09/04/birds-taxo-traits/

File: content/blog/2018-09-04-birds-taxo-traits.md [hunk 1]
```diff
@@ -0,0 +1,541 @@
...
+                                    "figure-markdown_github", "taxo.gif")) 
+```
+
+![animated tree with species names and order
+silhouette](/img/blog-images/2018-09-04-birds-taxo-traits/taxo.gif)
+
+This gif shows many species names and order giving us a feeling for what
+we might encounter in the county of Constance, but it lacks quantitative
+information about the species. It’d be interesting to create trees such
+as the ones of the [`metacorder`
```
Comment (sckott) [inline content/blog/2018-09-04-birds-taxo-traits.md]: metacoder

File: content/blog/2018-09-04-birds-taxo-traits.md [hunk 2]
```diff
@@ -0,0 +1,541 @@
...
+
+The script above isn’t the smartest since it doesn’t retain information
+about the species names. Not too bad here because I want to get a
+general idea of birds’ habitats and threats over the county.
+
+``` r
+habitat <- purrr::map_df(species_info, "habitat")
+```
+
+Out of 0, 203 are represented in this dataset.
```
Comment (sckott) [inline content/blog/2018-09-04-birds-taxo-traits.md]: > Out of 0, 203 are represented in this dataset.

i'm not sure this makes sense or maybe i'm reading it wrong?

File: content/blog/2018-09-04-birds-taxo-traits.md [hunk 3]
```diff
@@ -0,0 +1,541 @@
...
+
+| threat2                                | threat1                         |    n|
+|:---------------------------------------|:--------------------------------|----:|
+| Hunting & trapping terrestrial animals | Biological resource use         |   25|
+| Habitat shifting & alteration          | Climate change & severe weather |   24|
+| Renewable energy                       | Energy production & mining      |   16|
+| Agricultural & forestry effluents      | Pollution                       |   14|
+| Dams & water management/use            | Natural system modifications    |   14|
+
+It’s quite similar the ranking in this post on [BirdLife’s
```
Comment (sckott) [inline content/blog/2018-09-04-birds-taxo-traits.md]: qutie similar to?

File: content/blog/2018-09-04-birds-taxo-traits.md [hunk 4]
```diff
@@ -0,0 +1,541 @@
...
+input and output and was onboarded. Explore more of our packages suite,
+including and beyond the geospatial category,
+[here](https://ropensci.org/packages/).
+
+Mention `taxize` book?
+
+Guangchuang Yu’s other packages such as `ggtree` and `tidytree` are also
+of interest for manipulating and visualizing trees. More generally, the
+[Phylogenetics CRAN task
+view](https://cran.r-project.org/web/views/Phylogenetics.html) provides
```
Comment (sckott) [inline content/blog/2018-09-04-birds-taxo-traits.md]: there's a taxonomy task view as well but it's not on cran https://github.com/ropensci/taxonomy

File: content/blog/2018-09-04-birds-taxo-traits.md (hunk 2 above)
Comment (maelle) [reply content/blog/2018-09-04-birds-taxo-traits.md]: Oops buggy inline code

**Comment by maelle** [comment]
Thanks!

I re-organized the "further resources" section a bit, separating taxonomy from phylogenetics. I'm not sure how I could forget about the taxonomy CTV, very useful repo!

File: content/blog/2018-09-04-birds-taxo-traits.md [hunk 5]
```diff
@@ -0,0 +1,559 @@
...
+    variant: markdown_github
+    preserve_yaml: true
+---
+
+Thanks to the [second post of the series where we obtained data from
+eBird](https://ropensci.org/blog/2018/08/21/birds-radolfzell/) we know
+what birds were observed in the county of Constance. Now, not all
+species’ names mean a lot to me, and even if they did, there are a lot
+of them. In this post, we shall use rOpenSci’s packages accessing
+taxonomy and trait data in order to summarize the birds’ population of
```
Comment (stefaniebutland) [inline content/blog/2018-09-04-birds-taxo-traits.md]: phrase "to summarize the birds’ population of the county" is unclear. Do you mean "summarize the types of birds that populate the county"?

File: content/blog/2018-09-04-birds-taxo-traits.md [hunk 6]
```diff
@@ -0,0 +1,559 @@
...
+the county: armed with scientific and common names of birds, we have
+access to plenty of open data!
+
+### Getting and filtering the occurrence data
+
+For more details about the following code, refer to the [previous post
+of the series](https://ropensci.org/blog/2018/08/21/birds-radolfzell/).
+We only add a step to keep only data for the most recent years.
```
Comment (stefaniebutland) [inline content/blog/2018-09-04-birds-taxo-traits.md]: two "only"'s so maybe rephrase to:
We add one step to keep only data

File: content/blog/2018-09-04-birds-taxo-traits.md [hunk 7]
```diff
@@ -0,0 +1,559 @@
...
+``` r
+abundance <- dplyr::count(ebd, scientific_name)
+
+dico <- unique(dplyr::select(ebd, scientific_name,
+                             common_name))
+```
+
+### Getting taxonomic information
+
+In this section I would like to get an idea of how much of the bird
```
Comment (stefaniebutland) [inline content/blog/2018-09-04-birds-taxo-traits.md]: in "how much of the bird diversity" sounds specific

Do you mean "how much bird diversity" or "how diverse the types of birds are"?

File: content/blog/2018-09-04-birds-taxo-traits.md [hunk 8]
```diff
@@ -0,0 +1,559 @@
...
+library("ggplot2")
+ggtree::ggtree(tree$phylo) +
+    ggtree::geom_tiplab(aes(), size = 2, vjust=0.25) +
+  xlim(0, 150)
+```
+
+![labelled tree of all
+species](/img/blog-images/2018-09-04-birds-taxo-traits/firsttree-1.png)
+
+This tree is… unreadable. But at this point, it’s worth remembering that
```
Comment (stefaniebutland) [inline content/blog/2018-09-04-birds-taxo-traits.md]: I love this: "This tree is… unreadable. "

File: content/blog/2018-09-04-birds-taxo-traits.md [hunk 9]
```diff
@@ -0,0 +1,559 @@
...
+  dplyr::mutate(species = as.character(species)) %>%
+  dplyr::left_join(abundance,
+                   by = c("species" = "scientific_name"))
+
+ids <- purrr::map_df(orders$order, get_pic,          classification)
+save(ids, file = file.path("taxo", "ids.RData"))
+```
+
+It is rather tricky to automatically get pics from Phylopic since you
+might not get one for the order itself, maybe one for the subtaxa
```
Comment (stefaniebutland) [inline content/blog/2018-09-04-birds-taxo-traits.md]: should it be "subtaxon" in "maybe one for the subtaxa instead"?

File: content/blog/2018-09-04-birds-taxo-traits.md [hunk 10]
```diff
@@ -0,0 +1,559 @@
...
+```
+
+    ## [1] "/img/blog-images/2018-09-04-birds-taxo-traits/taxo.gif"
+
+``` r
+knitr::include_graphics(file.path("2018-09-04-birds-taxo-traits_files",
+                                    "figure-markdown_github", "taxo.gif")) 
+```
+
+![animated tree with species names and order
```
Comment (stefaniebutland) [inline content/blog/2018-09-04-birds-taxo-traits.md]: I love this!!! This is such a cool powerful illustration with birds beside phylo tree,  it immediately made me want to do this for some area of interest. Just realized I gotta share this post when up with the Dean of Science at our local university, Thompson Rivers University (TRU). He is a birder and a very tech friendly guy

Aside: In the `weathercan` post, Steffi LaZerte used her (bird) `feedr` package that she developed while at TRU in  Kamloops

File: content/blog/2018-09-04-birds-taxo-traits.md [hunk 11]
```diff
@@ -0,0 +1,559 @@
...
+silhouette](/img/blog-images/2018-09-04-birds-taxo-traits/taxo.gif)
+
+This gif shows many species names and order giving us a feeling for what
```
Comment (stefaniebutland) [inline content/blog/2018-09-04-birds-taxo-traits.md]: "many species names and order"
can you clarify what you mean by "order" here? 

should "Order" be capitalized in post when referring to taxonomic classification to avoid confusion?
I see an example here "[Homo sapiens is the most abundant member of the Order Primates](https://sciencemadeeasy.kinja.com/how-to-correctly-write-about-taxonomic-nomenclature-1623703107)"

File: content/blog/2018-09-04-birds-taxo-traits.md [hunk 12]
```diff
@@ -0,0 +1,559 @@
...
+we might encounter in the county of Constance, but it lacks quantitative
+information about the species. It’d be interesting to create trees such
+as the ones of the [`metacoder`
+package](https://github.com/grunwaldlab/metacoder) to reflect abudance,
```
Comment (stefaniebutland) [inline content/blog/2018-09-04-birds-taxo-traits.md]: s/abudance/abundance/

File: content/blog/2018-09-04-birds-taxo-traits.md [hunk 13]
```diff
@@ -0,0 +1,559 @@
...
+    ##  $ timing  : chr  "Agriculture & aquaculture" "Biological resource use" "Biological resource use" "Biological resource use" ...
+    ##  $ scope   : chr  "Annual & perennial non-timber crops" "Hunting & trapping terrestrial animals" "Hunting & trapping terrestrial animals" "Logging & wood harvesting" ...
+    ##  $ severity: chr  "Ongoing" "Ongoing" "Past, Likely to Return" "Ongoing" ...
+    ##  $ impact  : chr  "Ongoing" "Ongoing" "Past, Likely to Return" "Ongoing" ...
+
+As you can see, it is a depressing dataset since most threats are
+human-made, but such information can help conservation! Instead of
+diving into the fate of particular species, let’s get a general picture.
+There are 211 species, for 68 we get an entry in the threats data. What
+are the most common threats for them, now and in the future?
```
Comment (stefaniebutland) [inline content/blog/2018-09-04-birds-taxo-traits.md]: I got slowed down trying to interpret headings threat2 and thread1. If they are now and in the future could you write "now (THREAT2) and in the future (THREAT1)" or similar?

File: content/blog/2018-09-04-birds-taxo-traits.md (hunk 5 above)
Comment (maelle) [reply content/blog/2018-09-04-birds-taxo-traits.md]: I made it "in order to summarize some characteristics of the birds' population"

File: content/blog/2018-09-04-birds-taxo-traits.md (hunk 6 above)
Comment (maelle) [reply content/blog/2018-09-04-birds-taxo-traits.md]: ok "The single difference is our adding a step to keep only data for the most recent years."

File: content/blog/2018-09-04-birds-taxo-traits.md (hunk 7 above)
Comment (maelle) [reply content/blog/2018-09-04-birds-taxo-traits.md]: ok "how diverse the types of birds are"

File: content/blog/2018-09-04-birds-taxo-traits.md (hunk 8 above)
Comment (maelle) [reply content/blog/2018-09-04-birds-taxo-traits.md]: Honesty FTW :joy:

File: content/blog/2018-09-04-birds-taxo-traits.md (hunk 9 above)
Comment (maelle) [reply content/blog/2018-09-04-birds-taxo-traits.md]: indeed!

File: content/blog/2018-09-04-birds-taxo-traits.md (hunk 10 above)
Comment (maelle) [reply content/blog/2018-09-04-birds-taxo-traits.md]: Thanks! It's as kitsch as a wordbird hehe.

I can't think of a way to plug Steffi's packages `feedr` and `cavityuse` in the series, but if you think of one, please tell me.

File: content/blog/2018-09-04-birds-taxo-traits.md (hunk 11 above)
Comment (maelle) [reply content/blog/2018-09-04-birds-taxo-traits.md]: Added a link to https://en.wikipedia.org/wiki/Category:Bird_orders but "order" is not capitalized there? I can still capitalize if you prefer though.

File: content/blog/2018-09-04-birds-taxo-traits.md (hunk 13 above)
Comment (maelle) [reply content/blog/2018-09-04-birds-taxo-traits.md]: I'm renaming the headers
